src/main/scala/de/upb/cs/uc4/hyperledger/utilities/EnrollmentManager.scala

```diff
@@ -13,31 +13,31 @@ object EnrollmentManager {
     val wallet = WalletManager.getWallet(walletPath)
 
     // check if user already exists in my wallet
-    if (wallet.list.contains(username)) {
-      println(s"An identity for the user $username already exists in the wallet.")
+    if (WalletManager.containsIdentity(wallet, username)) {
+      println(s"[DEBUG] :: An identity for the user $username already exists in the wallet.")

```

Comment:
You should think about a Logger.

Reply:
Issue created

https://github.com/upb-uc4/hlf-api/issues/39